### ISSUE TITLE
Sonatype nexus complains about missing javadoc.jar 

### ISSUE THREAD

**Issue by dyu** [mjiderhamn/promote-maven-plugin#1]
```
Event: Failed: Javadoc Validation
Saturday, May 21, 2016 03:13:40 PHT (GMT+0800)
typeId  javadoc-staging

failureMessage  Missing: no javadoc jar found in folder '/com/dyuproject/protostuff/protostuff-json/1.0.10'
```

https://github.com/dyu/protostuff-1.0.x

Is there a config that would upload the javadoc jars as well?

**Comment by mjiderhamn** [comment]
Hi David. Did you generate the JavaDoc jar while building the SNAPSHOT build...?

I probably need to clarify the documentation that all artifacts that are to be released must be part of the build where `make-promotable` goal is invoked. I should also consider making `make-promotable` incremental, so that you can do `javadoc:jar`as part of the promotion.

**Comment by mjiderhamn** [comment]
I just released version 1.1.0 that allows for incremental artifact addition. That means you can do

```
mvn clean package promote:make-promotable
mvn javadoc:jar promote:make-promotable
mvn source:jar promote:make-promotable
```

and have all artifacts included when releasing.
